.env.example

```diff
@@ -1,17 +1,18 @@
-MONGO_URL=mongodb+srv://user-read-write1:sfd$42A3a@horondi-yl27g.mongodb.net/horondi-21?retryWrites=true&w=majority
```

Comment:
it is .env example. I removed  this string(left only name of variable) from it because in string are shown username and password for connection to database

Reply:
Provide here some dummy username, password and hostname